src/Test/Rexl.Bind.Test/Scripts/Binder/Functions/String.txt

```diff
@@ -238,6 +238,14 @@ Text.Replace("ABC", "X", s)
 Text.Replace("ABC", "B", "X")
 Text.Replace("ABACDAE", "A", "!!")
 
```

Comment:
There should be multiple kinds of test cases:
* Test with the named globals to ensure that errors are produced on bad input types, conversions are inserted where needed (eg, using `u2` for the length), arity is handled correctly.
* Test with special values to ensure that reduction happens as expected (once `ReduceCore` is implemented). Use enough cases that all the code in `ReduceCore` and in the `ExecXxx` methods is covered.
* Tests that use "opt" types in slots that lift over opt. Eg, tests that use `qi8` and something like `qu2` for the length.